setup.py

```diff
@@ -77,7 +77,7 @@
             'pytest-cov',
             'pytest-mypy',
             'pytest-flake8',
-            'pytest-benchmark',
+            'pytest-benchmark==3.1.1',
```

Comment:
I think we mean `'pytest-benchmark!=3.1.2'`

Reply:
Probably, yes. I'll revisit that soon. #123